### DISCUSSION TITLE
任务栏图标位置修改失效

### DISCUSSION THREAD

**Discussion by Norlan-Ch** [cgbsmy/TrayS#30] [Q&A]
任务栏图标位置修改失效该如何解决，如图所示，无论是选择中间还是右侧，任务栏位置始终停留在左侧
<img width="1113" alt="微信图片_20231120111417" src="https://github.com/cgbsmy/TrayS/assets/92942825/57a8c84f-3efe-43f7-889a-711e6df1e9cc">

**Comment by lin479909332** [comment]
我今天也突然失效了，有解决办法了吗？

**Reply by Norlan-Ch** [reply]
我的目前已经恢复正常了，但是由于时间有点久了我忘记是怎么恢复的了。试试更新和重启trays，以及重启电脑或任务栏（文件资源管理器）？
